corehq/apps/accounting/tests/utils.py

```diff
@@ -54,3 +62,31 @@ def teardown_subscription(cls, domain):
             cls.__accounts[domain].delete()
         finally:
             Subscription._get_active_subscription_by_domain.clear(Subscription, domain)
+
+
+@contextmanager
+def mocked_stripe_api_context(*args, **kwargs):
```

Comment:
What accept varargs? They appear to be unused.
```suggestion
def mocked_stripe_api():
```

And similar to Graham's comment, this can be used as either a decorator or a in a `with` statement, so the name of this function can be shortened and `mocked_stripe_api_decorator` can be removed.